pkg/memory/memory.go

```diff
@@ -37,8 +46,14 @@ type Area struct {
 	TotalPhysicalBytes int64 `json:"total_physical_bytes"`
 	TotalUsableBytes   int64 `json:"total_usable_bytes"`
 	// An array of sizes, in bytes, of memory pages supported in this area
-	SupportedPageSizes []uint64  `json:"supported_page_sizes"`
-	Modules            []*Module `json:"modules"`
+	SupportedPageSizes []uint64 `json:"supported_page_sizes"`
+	// Default system huge pages size, in bytes
+	DefaultHugePageSize uint64 `json:"default_huge_page_size"`
+	// Amount of memory, in bytes, consumed by huge pages of all sizes.
+	HugeTLBSize int64 `json:"huge_tlb_size"`
```

Comment:
the TLB is the lookaside buffer used for looking up the huge page physical location from a virtual address. It's also a Linux-specific thing and I'd prefer to keep the Area struct system-agnostic as much as possible.

How about we call this field `TotalHugePageBytes`?

Reply:
fixed